config/initializers/zz.rb

```diff
@@ -4,7 +4,7 @@
     [
       ActiveRecord::Base.send(:descendants).map(&:name),
       ActionController::Base.descendants,
-      (File.basename($0) != "rake" ? (defined?(Rake) && "rake") : nil),
+      # (File.basename($0) != "rake" ? (defined?(Rake) && "rake") : nil), # https://github.com/rails/rails/issues/25029
```

Comment:
don't want warn noise that we cannot do anything about ... just makes people learn to ignore stuff which I don't want :D